taxi/views.py

```diff
@@ -87,3 +89,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
all() is redundant, because it calls 2 sql queries: fetch all and then for each record fetch cars__manufacturer. It's better to call prefetch_related directly